# Message for publishing trajectories predicted for each tracked person
#

PredictedTrajectory[]   pred_trajs   # array of predicted trajectories
